pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -141,7 +151,7 @@ func (grm *nestedPendingOperations) Run(
 			return NewAlreadyExistsError(opKey)
 		}
 
-		backOffErr := previousOp.expBackoff.SafeToRetry(opKey.String())
+		backOffErr := previousOp.expBackoff.SafeToRetry(fmt.Sprintf("%+v", opKey))
```

Comment:
OMG can we include the exponential backoff package in the refactor? I just read why you even need to pass this string in.